aws-cloudformation/startup-advanced.sh

```diff
@@ -83,6 +83,13 @@ export GITHUB_TEAM_SLUG=$(aws cloudformation describe-stacks --stack-name=${STAC
 # attach volume if exist
 if [[ "${VOLUME_ID}" != "" ]]
 then
+    # wait volume "available" status
+    while true
```

Comment:
When using and endless loop you need a condition to break out of it, e.g., a maximum number of attempts.
This is already in place below and should be used here as well.

Reply:
Added the maximum attempts check.